docs/Basics-Layout.md

```diff
@@ -63,7 +63,7 @@ class AwesomeProject {
   }
 };
 
-AppRegistry.registerComponent('AwesomeProject', () => AwesomeProject);
+AppRegistry.registerComponent('AwesomeProject', () => JustifyContentBasics);
```

Comment:
I just love the name duck punch -- so doing that sounds awesome.

But I am for whatever can be done quickest for now.